__tests__/database/schema/UserSchema.test.ts

```diff
@@ -470,7 +470,8 @@ describe('TESTING UserSchema', () => {
             name: 'username',
             last_active: 'lastActiveAt',
             created_at: 'createdAt',
-            image: 'profilePicture'
+            image: 'profilePicture',
+            username: 'username'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The 'username' field is duplicated in the mapping. This could lead to unexpected behavior or errors. Please ensure that each key in the object is unique.
```diff
             name: 'username',
             last_active: 'lastActiveAt',
             created_at: 'createdAt',
-            image: 'profilePicture',
-            username: 'username'
+            image: 'profilePicture'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->